src/app/app.component.html

```diff
@@ -3,13 +3,13 @@
 
     <div class="m-0 p-0">
       <nav class="nav flex-column">
-        <a class="nav-brand mx-auto mb-1" routerLink=""><img src="assets/images/logo/airsonic-light-350x100.png"></a>
+        <a class="nav-brand mx-auto mb-1" routerLink=""><img src="./../assets/images/logo/airsonic-light-350x100.png"></a>
       </nav>
       <nav class="nav flex-column">
         <a class="nav-link text-white p-1" routerLink=""><i class="material-icons mx-3">music_note</i>{{ 'app.playing' | translate }}</a>
         <a class="nav-link text-white p-1" routerLink=""><i class="material-icons mx-3">star</i>{{ 'app.starred' | translate }}</a>
         <a class="nav-link text-white p-1" routerLink=""><i class="material-icons mx-3">playlist_play</i>{{ 'app.playlists' | translate }}</a>
-        <a class="nav-link text-white p-1" routerLink=""><i class="material-icons mx-3">settings</i>{{ 'app.settings' | translate }}</a>
+        <a class="nav-link text-white p-1" routerLink="/settings/user"><i class="material-icons mx-3">settings</i>{{ 'app.settings' | translate }}</a>
```

Comment:
If we can avoid as much as possible to use `/` in the router links it would be great.